docs/architecture/CONFIGURATION_ARCHITECTURE.md

```
@@ -0,0 +1,653 @@
+    skill: ChannelSkillConfigSchema,
+    code_tool_hook: ChannelCodeToolHookConfigSchema,
+    model_training: ChannelModelTrainingConfigSchema,
+  })),
+});
+
+export type ActivationConfig = Static<typeof ActivationConfigSchema>;
+```
+
+### 4.2 加载流程
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)